src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -131,23 +130,24 @@ public void Start()
             process.OutputDataReceived += (sender, args) =>
             {
                 string data = args.Data;
-                if (data != null)
+                if (data != null && data.Contains("Server started"))
                 {
-                    if (data.Contains("Server started"))
```

Comment:
Miscellaneous: combined `if` conditions.

Reply:
this introduces a change of behavior: previously, `WritelnToPane(data);` would have happened even if `!data.Contains("Server started")`, but now it will happen only if it contains `Server started` - is this intended?

Reply:
Good point; no, it wasn't. I'll separate the `if`s.